ext/hash/hash.stub.php

```diff
@@ -70,28 +70,28 @@ function hash_equals(#[\SensitiveParameter] string $known_string, #[\SensitivePa
 function hash_hkdf(string $algo, #[\SensitiveParameter] string $key, int $length = 0, string $info = "", string $salt = ""): string {}
 
 #ifdef PHP_MHASH_BC
-/** @deprecated */
+#[\Deprecated(since: '8.1')]
```

Comment:
Should we indicate to use the "proper" `hash_*` functions here?

Reply:
I briefly considered that, but the replacements are not really obvious and generic pointers are not particularly helpful.

As an example, I don't believe there is a replacement for `mhash_get_block_size()`. The naming is not particularly clear. Does the function return the block size or the output size? If the latter, then the replacement is “call `strlen()`”.

Reply:
That is fair. :)